indexing-service/src/main/java/org/apache/druid/indexing/overlord/supervisor/SupervisorResource.java

```diff
@@ -395,6 +395,25 @@ public Response shutdown(@PathParam("id") final String id)
     return terminate(id);
   }
 
+  @POST
+  @Path("/{id}/taskGroups/restart")
```

Comment:
```suggestion
  @Path("/{id}/taskGroups/handoff")
```

^ maybe this is a good name to match the function name.

Can you also please add javadocs for this function explaining that it is best effort. I think it is worth calling out that if a user passes in a taskGroup that does not exist, it will not error here.

And some user visible docs here https://github.com/apache/druid/blob/master/docs/api-reference/supervisor-api.md